enterprise/frontend/src/embedding-sdk/test/server-mocks/sdk-init.ts

```diff
@@ -26,7 +26,8 @@ import { createMockState } from "metabase-types/store/mocks";
 export const setupSdkState = ({
   currentUser = createMockUser(),
   settingValues = createMockSettings(),
-  tokenFeatures = createMockTokenFeatures(),
+  // TODO: change this to "embedding_sdk" once the token feature PR landed.
+  tokenFeatures = createMockTokenFeatures({ embedding: true }),
```

Comment:
Without the token features, the SDK test will fail due to the license check in the SDK.